fedoracommunity/connectors/bodhiconnector.py

```diff
@@ -525,15 +525,24 @@ def query_active_releases(self, filters=None, **params):
             filters = dict()
         filters = self._query_updates_filter.filter(filters, conn=self)
         package = filters.get('package')
-        pkgdb = get_connector('pkgdb')
         koji = get_connector('koji')._koji_client
         koji.multicall = True
 
-        # TODO - the list of Fedora releases can be obtained from the Bodhi API, instead of the pkgdb API.
-        # See https://bodhi.fedoraproject.org/releases/
-        for release in pkgdb.get_fedora_releases():
-            tag = release[0]
-            name = release[1]
+        releases_obj = self.call('releases', {})
+        total_pages = releases_obj['pages']
+        releases_all = releases_obj['releases']
+        if(total_pages > 1):
+            for i in range(2,total_pages+1):
+                temp = self.call('releases?page='+str(i), {})['releases']
+                releases_all.extend(temp)
+        releases_all.append({'dist_tag': 'rawhide',
```

Comment:
You'll need to filter these out in Python I believe because I don't see any way to show only the current releases in the Bodhi API docs:
https://bodhi.fedoraproject.org/docs/server_api/releases.html#service-1-GET